CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.10.0
+
+## Features ✨ and improvements 🏁
+* Adding new marker styling option: rotationAlignment: 'horizon' allowing marker rotation to match the curvature of the horizon in globe view ([#11894](https://github.com/mapbox/mapbox-gl-js/pull/11894))
+* Improve rendering performance on globe view ([#12050](https://github.com/mapbox/mapbox-gl-js/pull/12050))
+* Improve tile loading performance on low zoom levels ([#12061](https://github.com/mapbox/mapbox-gl-js/pull/12061))
+* Improve globe-mercator transition and map load performance with globe projection ([#12039](https://github.com/mapbox/mapbox-gl-js/pull/12039))
+* Improve panning precision on Globe View and relax constrains on lower zoom levels ([#12114](https://github.com/mapbox/mapbox-gl-js/pull/12114))
+
+
+## Bug fixes 🐞
+* Fix potential flickering on on image source updates ([#11928](https://github.com/mapbox/mapbox-gl-js/pull/11928))
+* Copy stylesheet to allow toggling different styles using setStyle without overwriting some of the properties ([#11942](https://github.com/mapbox/mapbox-gl-js/pull/11942))
+* Fix a bug where id expression didn't work for 0 values ([#12000](https://github.com/mapbox/mapbox-gl-js/pull/12000))
+* Fix precision errors in depth pack/unpack ([#12005](https://github.com/mapbox/mapbox-gl-js/pull/12005))
+* Allow style spec to use fog default values when toggling styles  ([#11992](https://github.com/mapbox/mapbox-gl-js/pull/11992))
```

Comment:
I'm pretty sure the copy the stylesheet to allow toggling setStyle and fog default values when toggling setStyle were in the 2.9.0 release. This release only includes this setStyle fix here: https://github.com/mapbox/mapbox-gl-js/pull/12049

Reply:
Confirmed #11992 and #11942 were in 2.9.0 so they can get removed from the changelog. Included in this cherry pick: https://github.com/mapbox/mapbox-gl-js/pull/11979

Reply:
Good catch @avpeery! This appears in the changelog [below ](https://github.com/mapbox/mapbox-gl-js/blob/main/CHANGELOG.md?plain=1#L30) for 2.9.0.
```suggestion
```